meilies/src/stream/stream.rs

```diff
@@ -63,18 +70,20 @@ impl fmt::Debug for Stream {
 
 impl fmt::Display for Stream {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self.from {
-            StartReadFrom::EventNumber(number) => write!(f, "{}:{}", self.name, number),
-            StartReadFrom::End => write!(f, "{}", self.name),
+        match (self.to, self.from) {
```

Comment:
Wouldn't it be more clear if you match in the other order (i.e. `(self.from, self.to)`) ?